recipes/garnett/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "garnett" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "9be24f8332757c00945849784316b06a2db58f06569a988bfb88c0af6455df1f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://glotzerlab.engin.umich.edu/Downloads/{{ name }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-cache-dir -vvv" 
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - numpy =1.11
```

Do you really need to be too restrictive to ``numpy`` ? or it can be greater or equal than?
```suggestion
    - numpy >=1.11
```

---

I think this is trhe default for host